dask/dataframe/core.py

```diff
@@ -7203,7 +7179,7 @@ def new_dd_object(dsk, name, meta, divisions, parent_meta=None):
         return get_parallel_type(meta)(dsk, name, meta, divisions)
 
 
-def partitionwise_graph(func, name, *args, **kwargs):
+def partitionwise_graph(func, op_name, *args, **kwargs):
```

Comment:
Is this argument name change necessary? It's not really any more or less clear to me than it was before.

Reply:
That makes a lot of sense. I guess we _could_ do 
```suggestion
def partitionwise_graph(func, name, /, *args, **kwargs):
```
so `name` can only be given positionally—I don't actually know if that would fix the problem though.

Just changing the parameter name to something less common is very reasonable. Maybe `out_name` or `df_name` instead? `op_name` to me makes me imagine `op = getattr(..., op_name)` happening somewhere, like you're giving the name of a pandas function or something.

Reply:
Okay - I can see the concern with `op_name`.  Maybe `layer_name` (since that is what it is)?

Reply:
Sure!